client/components/editor/Sidebar/Navigation.vue

```diff
@@ -11,13 +11,23 @@
         class="my-3 mx-1" />
     </v-sheet>
     <v-treeview
-      @update:active="navigateTo"
+      ref="activityTree"
```

Comment:
The issue with an empty message is that the `v-treeview` component does not have native support for it. Does not have any API capabilities to notify the parent component so there is no clean way of knowing if the tree view is empty or not. One of the ways that I found to do it is to watch its computed property like so:
```js
mounted() {
  this.$watch(
    () => this.$refs.activityTree.excludedItems.size,
    val => (this.showEmptyMessage = this.totalSize === val)
  );
}
```
but I am not really a fan of watching properties of components such as these that are not exposed to the outside since they can change in minor versions even and break the app. If you are ok with this type of approach I will continue with this approach.

Reply:
Found another way